packages/client/hmi-client/src/components/workflow/ops/optimize-ciemss/tera-optimize-ciemss.vue

```diff
@@ -723,6 +824,64 @@ onMounted(async () => {
 	initialize();
 });
 
+const preparedInterventionsCharts = computed(() => {
+	const state = cloneDeep(props.node.state);
```

Comment:
A few things to say here, applies to both "preparedInterventionsCharts" and "preparedCharts"?

- What is going on with all the cloning? What are we trying to do?
- If possible, stick to either working with "state" or working with "knobs", it makes reading more consistent even though they may be pointing to the same value.
- Both functions (optimization-chart and intervention-chart) are returning createOptimizeForecastChart ... is that correct? Shouldn't intervention have its own chart, like ... you have a "tera-intervention-chart.vue"??